client/me/help/main.jsx

```diff
@@ -114,33 +113,20 @@ class Help extends React.PureComponent {
 		);
 	};
 
-	getSupportLinks = () => {
-		return (
+	getSupportLinks = () => (
+		<>
+			<h2 className="help__section-title">{ this.props.translate( 'More Resources' ) }</h2>
 			<div className="help__support-links">
-				<CompactCard
-					className="help__support-link"
-					href={ localizeUrl( 'https://wordpress.com/support/' ) }
-					target="__blank"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">
-							{ this.props.translate( 'All support articles' ) }
-						</h2>
-						<p className="help__support-link-content">
-							{ this.props.translate(
-								'Looking to learn more about a feature? Our docs have all the details.'
-							) }
-						</p>
-					</div>
-				</CompactCard>
+				{ this.getCoursesTeaser() }
 				<CompactCard
 					className="help__support-link"
 					href={ localizeUrl( 'https://wordpress.com/support/video-tutorials/' ) }
 					target="__blank"
 				>
+					<Gridicon icon="video" size={ 36 } />
 					<div className="help__support-link-section">
 						<h2 className="help__support-link-title">
-							{ this.props.translate( 'Quick help video tutorials' ) }
+							{ this.props.translate( 'Video tutorials' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.